src/components/CytoscapeLayout/__tests__/CytoscapeLayout.test.tsx

```diff
@@ -1,41 +1,55 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
-import { Button } from 'patternfly-react';
+// import { Button } from 'patternfly-react';
```

Comment:
If this is unused may as well remove it.

Reply:
noted